### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by jacobwilki01**

### CODE REVIEW

File: CardsCashCasino/Manager/TexasHoldEmManager.cs [hunk 1]
```diff
@@ -194,11 +194,6 @@ public class TexasHoldEmManager
         /// </summary>
         private PokerActionButton? _allInButton;
         #endregion
-        
-        /// <summary>
-        /// The community cards shared by all players.
-        /// </summary>
-        private List<Card> _communityCards = new();
```
Comment (modmo22) [inline CardsCashCasino/Manager/TexasHoldEmManager.cs:201]: Is deleting this intentional?

File: UnitTests/BlackjackTests.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+            _bettingManager.LoadContent();
+        }
+
+        [TestMethod]
+        public void TestDoubleDown()
+        {
+            _blackjackManager.StartGameWithoutUI(50);
+            _bettingManager.ConfirmBetWithoutUI(50);
+
+            int prevBet = 0 + BettingManager.UserBet;
```
Comment (modmo22) [inline UnitTests/BlackjackTests.cs:37]: This should be `_bettingManager` and not `BettingManager`, right?

File: UnitTests/BlackjackTests.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+            _blackjackManager.CallActionWithoutUI(2); // call double down.
+
+            Assert.AreEqual(prevBet * 2, BettingManager.UserBet);
```
Comment (modmo22) [inline UnitTests/BlackjackTests.cs:41]: And same here?

**Review by modmo22 — CHANGES_REQUESTED** [review]

File: UnitTests/BlackjackTests.cs (hunk 2 above)
Comment (jacobwilki01) [reply UnitTests/BlackjackTests.cs:37]: No. Its a reference to a static integer. We did this so it was a global value set throughout the code.

File: CardsCashCasino/Manager/TexasHoldEmManager.cs (hunk 1 above)
Comment (jacobwilki01) [reply CardsCashCasino/Manager/TexasHoldEmManager.cs:201]: Yes. There were two instances of this list. Likely the result of a git merge failure or someone not running their code before pushing.

File: UnitTests/BlackjackTests.cs (hunk 3 above)
Comment (jacobwilki01) [reply UnitTests/BlackjackTests.cs:41]: Yes. See above.

**Review by modmo22 — APPROVED** [review]

**Review by eBerkley — APPROVED** [review]

**Pull request merged**